src/main/java/uk/gov/hmcts/reform/sendletter/services/LetterService.java

```diff
@@ -47,49 +48,48 @@ public LetterService(LetterRepository letterRepository, Zipper zipper, ObjectMap
 
     @Transactional
```

Comment:
Looks like both methods are wrappers/helpers of generic `send(LetterInfo, Supplier)` method. Maybe worth just moving `@Transactional` to that method instead?